src/_zkapauthorizer/tests/test_plugin.py

```diff
@@ -108,6 +110,58 @@ def get_rref(interface=None):
     return LocalRemote(DummyReferenceable(interface))
 
 
+class OpenStoreTests(TestCase):
+    @skipIf(platform.isWindows(), "Hard to prevent directory creation on Windows")
+    @given(tahoe_configs(), datetimes())
+    def test_uncreateable_store_directory(self, get_config, now):
+        """
+        If the underlying directory in the node configuration cannot be created
+        then ``open_store`` raises ``StoreOpenError``.
+        """
+        nodedir = FilePath(self.useFixture(TempDir()).join("node"))
```

Comment:
This looks like a clue about codecov: it maybe only looks at "windows" coverage? (Since it says this test isn't run